### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Sorry, something on top of https://github.com/skrub-data/skrub/pull/967 : a few typos, a bit of layout (the "|" adds vertical space), and an additional line which I believe helps.

### CODE REVIEW

File: examples/01_encodings.py [hunk 1]
```diff
@@ -62,11 +62,11 @@
 # set of columns: some are numerical (``'year_first_hired'``), some dates
```
Comment (TheooJ) [inline examples/01_encodings.py:62]: some are dates ?

Comment (GaelVaroquaux) [reply examples/01_encodings.py:62]: Actually, I think that this is proper English

**Comment by TheooJ** [comment]
Should it be `skrub`'s ? or do we keep skrub's ?

**Comment by GaelVaroquaux** [comment]
> Should it be `skrub`'s ? or do we keep skrub's ?

It doesn't matter much to me

**Review by TheooJ — APPROVED** [review]
Merging then !

**Comment by GaelVaroquaux** [comment]
Thanks!!

**Pull request merged**